conf/messages.en

```diff
@@ -497,19 +497,12 @@ registrationConfirmation.panel.p1 = Your digital platform reporting user ID is {
 registrationConfirmation.person.p1.0 = You have
 registrationConfirmation.organisation.p1.0 = {0} has
 registrationConfirmation.p1.1 = successfully registered for digital platform reporting on {0}.
-registrationConfirmation.p2.0 = Make a note of your digital platform reporting user ID.
-registrationConfirmation.p2.1 = Make a note of your digital platform reporting user ID. We have sent a confirmation email to {0}.
-registrationConfirmation.h2.1 = What you need to do next
-registrationConfirmation.thirdParty.p3 = To send a submission, you must add the platform operators you will report for.
-registrationConfirmation.organisation.p3 = To send a submission, you must add your business as a platform operator.
-registrationConfirmation.h2.2 = Do you want to add a platform operator now?
-registrationConfirmation.p4 = You can add a platform operator at any time.
+registrationConfirmation.p2 = Make a note of your digital platform reporting user ID. We have sent a confirmation email to {0}.
```

Comment:
Should there be a full stop straight after the email address ?

Reply:
I have not changed that. This is how it was before. I only changed the key. It was tested before, so Yes, I think there should be